src/contracts/core/DelegationManager.sol

```diff
@@ -311,117 +331,106 @@ contract DelegationManager is
+     * @param strategy The strategy in which to decrease the delegated scaled shares.
+     * @param removedShares The number of shares to decremented for the strategy in the
+     * StrategyManager/EigenPodManager
      *
-     * @dev *If the staker is actively delegated*, then decreases the `staker`'s delegated shares in `strategy` by `shares`. Otherwise does nothing.
+     * @dev *If the staker is actively delegated*, then decreases the `staker`'s delegated delegatedShares in `strategy` by `scaledShares`. Otherwise does nothing.
      * @dev Callable only by the StrategyManager or EigenPodManager.
      */
     function decreaseDelegatedShares(
         address staker,
         IStrategy strategy,
-        uint256 shares
+        uint256 removedShares
     ) external onlyStrategyManagerOrEigenPodManager {
         // if the staker is delegated to an operator
-        if (isDelegated(staker)) {
-            address operator = delegatedTo[staker];
+        // if (isDelegated(staker)) {
+        //     address operator = delegatedTo[staker];
 
-            // forgefmt: disable-next-item
-            // subtract strategy shares from delegate's shares
-            _decreaseOperatorShares({
-                operator: operator, 
-                staker: staker, 
-                strategy: strategy, 
-                shares: shares
-            });
-        }
-    }
+        //     uint64 totalMagnitude = allocationManager.getTotalMagnitude(operator, strategy);
 
-    /**
-     * @notice Owner-only function for modifying the value of the `minWithdrawalDelayBlocks` variable.
-     * @param newMinWithdrawalDelayBlocks new value of `minWithdrawalDelayBlocks`.
-     */
-    function setMinWithdrawalDelayBlocks(uint256 newMinWithdrawalDelayBlocks) external onlyOwner {
-        _setMinWithdrawalDelayBlocks(newMinWithdrawalDelayBlocks);
-    }
-
-    /**
-     * @notice Called by owner to set the minimum withdrawal delay blocks for each passed in strategy
-     * Note that the min number of blocks to complete a withdrawal of a strategy is
-     * MAX(minWithdrawalDelayBlocks, strategyWithdrawalDelayBlocks[strategy])
-     * @param strategies The strategies to set the minimum withdrawal delay blocks for
-     * @param withdrawalDelayBlocks The minimum withdrawal delay blocks to set for each strategy
-     */
-    function setStrategyWithdrawalDelayBlocks(
-        IStrategy[] calldata strategies,
-        uint256[] calldata withdrawalDelayBlocks
-    ) external onlyOwner {
-        _setStrategyWithdrawalDelayBlocks(strategies, withdrawalDelayBlocks);
+        //     // subtract strategy shares from delegated scaled shares
+        //     _decreaseOperatorScaledShares({
+        //         operator: operator,
+        //         staker: staker,
+        //         strategy: strategy,
+        //         shares: removedShares,
+        //         totalMagnitude: totalMagnitude
+        //     });
+        // }
```

Comment:
should probably at least have a TODO tag or similar for this totally commented-out function (unless it is being eliminated?)

Reply:
Updated, we have `_decreaseDelegation` now